loki/client.go

```diff
@@ -162,7 +162,7 @@ func NewWithLogger(cfg Config, logger log.Logger) (*Client, error) {
 		return nil, err
 	}
 
-	c.client, err = config.NewClientFromConfig(cfg.Client, "promtail", false, false)
+	c.client, err = config.NewClientFromConfig(cfg.Client, "promtail", config.WithHTTP2Disabled())
```

Comment:
I left HTTP2 disabled as that was the previous behavior, but in newer versions of Go could this be enabled again?